2013/02/22 17:57:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/
---
2013/02/22 17:58:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/
---
2013/02/22 18:02:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/index.php?r=site/index
---
2013/02/22 18:29:42 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/22 18:30:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/22 18:38:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hello".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hello')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/hello
---
2013/02/22 18:40:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "page/about".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('page/about')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/page/about
---
2013/02/24 11:18:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:18:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:19:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:21:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/site/hello
---
2013/02/24 11:30:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 11:31:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 11:31:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 12:17:20 [error] [php] include(cHtml.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 E:\xampp_new\htdocs\YiiDemo\protected\views\website\index.php(10): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\web\CBaseController.php(126): require()
#2 E:\xampp_new\htdocs\yii\framework\web\CBaseController.php(95): WebsiteController->renderInternal()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(869): WebsiteController->renderFile()
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(782): WebsiteController->renderPartial()
#5 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(12): WebsiteController->render()
#6 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionIndex()
#7 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): WebsiteController->runAction()
#9 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): WebsiteController->runActionWithFilters()
#10 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): WebsiteController->run()
#11 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/website
in E:\xampp_new\htdocs\YiiDemo\protected\views\website\index.php (10)
in E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php (12)
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/24 12:28:49 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/n...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/index/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website/index
---
2013/02/24 12:30:36 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/n...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/index/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website/index
---
2013/02/24 12:30:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website
---
2013/02/24 12:30:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "duc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('duc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('duc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/duc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/duc
---
2013/02/24 12:31:05 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/
---
2013/02/24 12:31:21 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/d...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/index/duc
---
2013/02/24 12:44:22 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/index.php/website/index/name/Duc
---
2013/02/24 12:44:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/index.php/website/index/name/Duc
---
2013/02/24 12:45:04 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website
---
2013/02/24 12:46:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website
---
2013/02/24 12:46:11 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name
---
2013/02/24 12:46:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/name
---
2013/02/24 12:46:40 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:46:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:47:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:47:28 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:49:22 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 12:49:42 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 12:49:45 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 14:13:03 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 14:24:20 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/he...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page/hello
---
2013/02/24 14:24:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:26:32 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:27:55 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:28:30 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/he...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page/hello
---
2013/02/24 14:28:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 17:01:39 [error] [php] strpos() expects parameter 1 to be string, array given (E:\xampp_new\htdocs\yii\framework\web\CController.php:963)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionTest()
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): WebsiteController->runAction()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): WebsiteController->runActionWithFilters()
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): WebsiteController->run()
#5 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/website/test
in E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php (28)
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/24 17:02:19 [error] [exception.CException] exception 'CException' with message 'WebsiteController and its behaviors do not have a method or closure named "AbsoluteUrl".' in E:\xampp_new\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(29): CComponent->__call('AbsoluteUrl', Array)
#1 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(29): WebsiteController->AbsoluteUrl('website/page', Array)
#2 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionTest()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('test')
#7 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/test')
#8 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/website/test
---
2013/02/25 10:27:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "test".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/test
---
2013/02/25 17:48:27 [error] [exception.CException] exception 'CException' with message 'TestController cannot find the requested view "test".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('test', NULL, true)
#1 E:\xampp_new\htdocs\YiiDemo\protected\controllers\TestController.php(15): CController->render('test')
#2 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): TestController->actionIndex()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#8 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/test
---
2013/02/26 16:33:36 [error] [php] include(ModuleUrlManager.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\base\CComponent.php(556): call_user_func()
#2 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(196): CWebApplication->raiseEvent()
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:10:18 [error] [php] include(ModuleUrlManager.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\base\CComponent.php(556): call_user_func()
#2 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(196): CWebApplication->raiseEvent()
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:11:40 [error] [php] call_user_func() expects parameter 1 to be a valid callback, class 'ModuleUrlManager' not found (E:\xampp_new\htdocs\yii\framework\base\CComponent.php:556)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#1 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:12:43 [error] [php] call_user_func() expects parameter 1 to be a valid callback, class 'ModuleUrlManager' not found (E:\xampp_new\htdocs\yii\framework\base\CComponent.php:556)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#1 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/index.php/gii/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:14:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php:369
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 E:\xampp_new\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 E:\xampp_new\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 E:\xampp_new\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 E:\xampp_new\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 E:\xampp_new\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 E:\xampp_new\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/YiiDemo/gii/model
---
